Makefile

```diff
@@ -219,13 +219,31 @@ fleets-remove:
 
 fleets-update: fleets-remove $(FLEETS)
 
-rcc:
+UI_RESOURCES := resources.rcc
+
+# The list this generates is very large and causes make to take longer
+# comparing timestamps than it does to simply run the rcc target.
+
+# UI_SOURCES := $(shell find ui -type f | grep -v resources\.qrc)
+
+# This list is much smaller and the rcc target completes very quickly if
+# there's nothing to be done. However, if there's a change to e.g. a PNG file
+# then it will be necessary to delete resources.rcc (or touch a .qml file,
+# etc.) in order to trigger a fresh build of resources.rcc
+
+UI_SOURCES := $(shell find ui -type f \
+		| grep -E "qmldir|\.(qml|qrc)\$$" \
+		| grep -v resources\.qrc)
```

Comment:
You can do the same thing with just `find`, without having to invoke two more processes:
```sh
 > find ui -type f -regextype egrep -iregex '.*(qmldir|qml|qrc)$' -not -iname 'resources.qrc' | wc -l
484

 > find ui -type f | grep -E "qmldir|\.(qml|qrc)\$$" | grep -v resources.qrc | wc -l         
484
```

Reply:
Thanks, change made, though I did need to account for BSD `find` not having the `-regextype` option (it has `-E` instead).

Reply:
Oh, that's annoying.